<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelloMutex</class>
 <widget class="QDialog" name="HelloMutex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRC Check</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="keysightTabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mutex</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="txtIn"/>
          </item>
          <item>
           <widget class="QTextEdit" name="txtOut"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnThread">
              <property name="text">
               <string>Start thread</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTimer">
              <property name="text">
               <string>Start timer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnAppend">
              <property name="text">
               <string>Append</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="txtParse"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnStartParse">
           <property name="text">
            <string>Start Parse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStopParse">
           <property name="text">
            <string>Stop Parse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>keysightTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
